packages/fxa-auth-server/lib/senders/emails/layouts/subscription/index.mjml

```diff
@@ -7,7 +7,7 @@
     <mj-raw>
       <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
     </mj-raw>
-    <mj-title><%- subject %></mj-title>
+    <mj-title><%- locals.subject %></mj-title>
```

Comment:
Was that just broken before you added `locals.`?

Reply:
It doesn't look broken, but it was also the only spot where we didn't specify `locals.`... modified for consistency